### ISSUE TITLE
[UI] Helm chart feature request: Watch multi namespaces and parameterize teams

### ISSUE THREAD

**Issue by caiconkhicon** [zalando/postgres-operator#1373]
Please, answer some short questions which should help us to understand your problem / question better?

- **Which image of the operator are you using?** registry.opensource.zalan.do/acid/postgres-operator-ui:v1.6.1
- **Where do you run it - cloud or metal? Kubernetes or OpenShift?** Bare Metal K8s
- **Are you running Postgres Operator in production?** yes
- **Type of issue?** Feature request

I want to use Helm to deploy postgres-operator-ui (https://github.com/zalando/postgres-operator/blob/master/charts/postgres-operator-ui). However, it lacks of two features:
- Multi namespaces: my databases are deployed in many namespaces. ~With the current Helm chart, it is possible to watch one namespace only~. It is possible to watch all namespaces with `targetNamespace: *`. Please create a documentation for the `postgres-operator-ui` Helm chart to mention this.
- Parameterize team names: Team names are specified as a list. However, it is fixed with one item named `acid`. I have tested changing it manually and it worked. So please implement it. (https://github.com/zalando/postgres-operator/blob/master/charts/postgres-operator-ui/templates/deployment.yaml#L54-L58)

**Comment by caiconkhicon** [comment]
I created https://github.com/zalando/postgres-operator/pull/1375 to add the TEAMS feature

**Comment by yfoelling** [comment]
@caiconkhicon you can watch all namespaces by setting it to:
```  targetNamespace: "*"```

i found that out by trial-error, since i also did not find it in the documentation

**Comment by FxKu** [comment]
fyi we have a reference doc for all config options where this is explained: https://github.com/zalando/postgres-operator/blob/master/docs/reference/operator_parameters.md#kubernetes-resources as well as in our [admin](https://github.com/zalando/postgres-operator/blob/master/docs/administrator.md#specify-the-namespace-to-watch) docs.

**Comment by caiconkhicon** [comment]
@FxKu : I don't think that it's exactly correct. The documentation is for `postgres-operator`, not for the UI. The mentioned parameter is also `watched_namespace`, not `targetNamespace` for the UI.

**Comment by FxKu** [comment]
ah yes, true for UI there's not much documentation. But yes, it works with `targetNamespace: "*"` indeed. Not sure, if this should be the default.

**Comment by caiconkhicon** [comment]
I agree that it should not be `*` per default. However, it should be configurable. I will update the issue description to reflex that we need a doc for the UI about that `targetNamespace`.

**Comment by SNThrailkill** [comment]
the `targetNamespace` kept me from using this for weeks. Thankfully I knew this had to be a thing and found this issue. PLEASE add this parameter to the documentation.
